lib/modules/editor/widgets/transition_editor.dart

```diff
@@ -0,0 +1,166 @@
+        ? null
+        : () {
+            node.transitions.remove(value);
+            editor.translation.assets.remove(value.id);
+          },
+    builder: (_, setState) {
+      return [
+        if (node.transitionInputSource == TransitionInputSource.select)
+          ListTile(
+            title: TextFormField(
+              decoration: const InputDecoration(
+                labelText: 'Transition text',
+                prefixIcon: Icon(Icons.short_text_rounded),
+              ),
+              autofocus: true,
+              initialValue: translation.text,
+              validator: emptyValidator,
+              onChanged: (s) {
+                newText = s.trim();
+              },
+            ),
+          ),
+        buildExplanationTile(context, 'Target node'),
+        Provider.value(
+          value: editor,
+          child: NodeTile(
+            editor.story.nodes[result.targetNodeId],
+            onTap: () => pickNode(
+              context,
+              editor.story.nodes[result.targetNodeId],
+            ).then((r) {
+              if (r != null) {
+                setState(() {
+                  result.targetNodeId = r.id;
+                });
+              }
+            }),
+          ),
+        ),
+        if (node.transitionInputSource == TransitionInputSource.cell)
+          Provider.value(
+            value: editor,
+            builder: (context, _) {
+              final celled = result as CelledTransition;
+              void setMode(ComparisionMode? v) => setState(() {
+                    celled.comparision = v;
+                  });
+              return Column(
+                children: [
+                  buildExplanationTile(context, 'Cell-controlled logic'),
+                  CellTile(
+                    editor.story.cells[celled.targetCellId],
+                    onTap: () => pickCell(context).then(
+                      (v) => setState(() {
+                        celled.targetCellId = v?.id ?? celled.targetCellId;
+                      }),
+                    ),
+                  ),
+                  RadioListTile<ComparisionMode?>(
+                    value: null,
+                    groupValue: celled.comparision,
+                    title: const Text('Pass'),
+                    subtitle: const Text('Always true (use for default)'),
+                    onChanged: setMode,
+                  ),
+                  RadioListTile<ComparisionMode?>(
+                    value: ComparisionMode.equal,
+                    groupValue: celled.comparision,
+                    title: const Text('[=] Equal'),
+                    subtitle: const Text('Are the values exactly the same?'),
+                    onChanged: setMode,
+                  ),
+                  RadioListTile<ComparisionMode?>(
+                    value: ComparisionMode.lesser,
+                    groupValue: celled.comparision,
+                    title: const Text('[<] Lesser'),
```

Comment:
"Less" is already a comparative adjective. "Lesser" has a different tone that is mostly used condescendingly. (don't forget to fix the enum as well)